multimodal/src/autogluon/multimodal/data/dataset_mmlab/multi_image_mix_dataset.py

```diff
@@ -17,834 +17,823 @@
-
-            if "gt_masks" in results:
-                raise NotImplementedError("RandomAffine only supports bbox.")
-        return results
-
-    def __repr__(self):
-        repr_str = self.__class__.__name__
-        repr_str += f"(max_rotate_degree={self.max_rotate_degree}, "
-        repr_str += f"max_translate_ratio={self.max_translate_ratio}, "
-        repr_str += f"scaling_ratio_range={self.scaling_ratio_range}, "
-        repr_str += f"max_shear_degree={self.max_shear_degree}, "
-        repr_str += f"border={self.border}, "
-        repr_str += f"border_val={self.border_val}, "
-        repr_str += f"bbox_clip_border={self.bbox_clip_border})"
-        return repr_str
-
-    @staticmethod
-    def _get_rotation_matrix(rotate_degrees: float) -> np.ndarray:
-        radian = math.radians(rotate_degrees)
-        rotation_matrix = np.array(
-            [[np.cos(radian), -np.sin(radian), 0.0], [np.sin(radian), np.cos(radian), 0.0], [0.0, 0.0, 1.0]],
-            dtype=np.float32,
-        )
-        return rotation_matrix
-
-    @staticmethod
-    def _get_scaling_matrix(scale_ratio: float) -> np.ndarray:
-        scaling_matrix = np.array(
-            [[scale_ratio, 0.0, 0.0], [0.0, scale_ratio, 0.0], [0.0, 0.0, 1.0]], dtype=np.float32
-        )
-        return scaling_matrix
-
-    @staticmethod
-    def _get_shear_matrix(x_shear_degrees: float, y_shear_degrees: float) -> np.ndarray:
-        x_radian = math.radians(x_shear_degrees)
-        y_radian = math.radians(y_shear_degrees)
-        shear_matrix = np.array(
-            [[1, np.tan(x_radian), 0.0], [np.tan(y_radian), 1, 0.0], [0.0, 0.0, 1.0]], dtype=np.float32
-        )
-        return shear_matrix
-
-    @staticmethod
-    def _get_translation_matrix(x: float, y: float) -> np.ndarray:
-        translation_matrix = np.array([[1, 0.0, x], [0.0, 1, y], [0.0, 0.0, 1.0]], dtype=np.float32)
-        return translation_matrix
+            return rotation_matrix
+
+        @staticmethod
+        def _get_scaling_matrix(scale_ratio: float) -> np.ndarray:
+            scaling_matrix = np.array(
+                [[scale_ratio, 0.0, 0.0], [0.0, scale_ratio, 0.0], [0.0, 0.0, 1.0]], dtype=np.float32
+            )
+            return scaling_matrix
+
+        @staticmethod
+        def _get_shear_matrix(x_shear_degrees: float, y_shear_degrees: float) -> np.ndarray:
+            x_radian = math.radians(x_shear_degrees)
+            y_radian = math.radians(y_shear_degrees)
+            shear_matrix = np.array(
+                [[1, np.tan(x_radian), 0.0], [np.tan(y_radian), 1, 0.0], [0.0, 0.0, 1.0]], dtype=np.float32
+            )
+            return shear_matrix
+
+        @staticmethod
+        def _get_translation_matrix(x: float, y: float) -> np.ndarray:
+            translation_matrix = np.array([[1, 0.0, x], [0.0, 1, y], [0.0, 0.0, 1.0]], dtype=np.float32)
+            return translation_matrix
+
+except Exception as e:
```

Comment:
Why use `Exception` when we can be more specific such as `ImportError`?

Reply:
Addressed in #3277